### DISCUSSION TITLE
Native inlay hint

### DISCUSSION THREAD

**Discussion by jacknicklenson** [VonHeikemen/lsp-zero.nvim#323] [General]
Hi thx for the plugin,

Can you add setup instruction for neovim' s native inlay hint which is upcoming feature of neovim 0.10.

There's already instruction for lsp inlay hint plugin but there's no for native.

**Comment by VonHeikemen** [comment]
Is mostly the same thing. You enable inlay hints in your server's settings then call a function in the `on_attach` function.

Here is an example with lua_ls.

```lua
require('lspconfig').lua_ls.setup({
  on_attach = function(client, bufnr)
    vim.lsp.inlay_hint(bufnr, true)
  end,
  settings = {
    Lua = {
      hint = {
        enable = true,
      },
    },
  },
})
```

This can break at any moment. Neovim can change the api or it could have a bug in a specific version of nightly.
